config.default

```diff
@@ -76,5 +76,30 @@
         }
       }
     }
+  },
+  "repo_lists": {
+    "modules": [
```

Comment:
As for me, if we need the prefix (`repolist-`), we should require it be here explicitly. Otherwise it is easy to get convinced that RWS will serve corresponding requests on `/modules`, not `/repolist-modules`.

Reply:
I just find it more explicit and closer to resulting HTTP URIs.

I can imagine another implementatoin, where the anchors are arbitrary strings, not only `repolist-.*`. If the config format / validation rules would help me to spot the idea, I would have less chances to do a mistake.

Anyway, it is not where I want to impose on my opinion.

Reply:
Updated.